kube-monitoring/charts/templates/pmon.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ $.Release.Name }}-sd
```

Comment:
Does "sd" stand for "service discovery" here?

Reply:
yes